### PR TITLE
Docs/update tutorials

### PR DESCRIPTION
**Pull request by schrodit**
**How to categorize this PR?**

/area documentation
/priority normal

**What this PR does / why we need it**:
Add glossary and improve some documentation

Many thanks to @MrBatschner and @hendrikKahl 

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: docs/concepts/Glossary.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+#### _DeployExecutions_
+
+  A list of templates to generate [deploy items](#_deployitem_) as part of a [Blueprint](#_blueprint_). Mainly used to describe the installation instructions and customize it using the data provided as [imports](#_import_).
+
+#### _DeployItem_
+
+  A DeployItem is the interface between the Landscaper controller and the [Deployers](#_deployer_). It contains input data and a set of Deployer-specific instructions on how to install a component (e.g. install a helm chart with some custom values). Additionally, it is used to record the status as returned by the Deployer.
+
+#### _Execution_
```
Comment (Diaphteiros) [inline docs/concepts/Glossary.md:50]: It seems that something is missing here ...

**Review by hendrikKahl — APPROVED** [review]
👍

File: docs/concepts/Glossary.md (hunk 1 above)
Comment (schrodit) [reply docs/concepts/Glossary.md:50]: the glossary ist just not yet completed.
But we rather wanted to push the changes in and continuously improve missing pieces.

Comment (Diaphteiros) [reply docs/concepts/Glossary.md:50]: Ah, ok, then lgtm

**Review by Diaphteiros — APPROVED** [review]

**Pull request merged**